### DISCUSSION TITLE
Issues with Lighthouse Purchased from Bitcraze

### DISCUSSION THREAD

**Discussion by Will-Emery** [bitcraze/discussions#1112] [Q&A]
Hello, a while back the group that I'm working with purchased 2 Steam VR Base Station 2.0s from Bitcraze. To my knowledge, and my other colleagues, neither of them has been used. One of them works fine and we can see the IR light using cellphone cameras. The other doesn't fire up the IR emitter and has a red flashing light near the top of the unit. With this second base station, I can hear something spinning up inside of it, but the issue seems to be with the IR emitter.  From searching online I saw people recommending I reach out to Steam support, but figured this was a good place to ask for solutions to this problem.

**Comment by knmcguire** [comment]
Hi! 

You are at the right place since that seems like a clear hardware fault. Since you bought it from us, could you send us an email at contact _at_ bitcraze.io with the order number? Then we will send a replacement.
